src/meta_tests.rs

```diff
@@ -0,0 +1,115 @@
+#[cfg(test)]
+extern crate std;
+#[cfg(test)]
+use std::println;
+use std::time::{SystemTime, UNIX_EPOCH};
+
+use codec::Encode;
+use gstd::{ActorId, BTreeMap};
+use gtest::{Program, System};
+use lt_io::*;
+const USERS: &[u64] = &[3, 4, 5];
+
+fn init(sys: &System) {
+    sys.init_logger();
+
+    let lt = Program::current(sys);
+
+    let res = lt.send_bytes_with_value(USERS[0], b"Init", 10000);
+
+    assert!(res.log().is_empty());
+}
+
+#[test]
+fn meta_tests() {
+    let time = SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .expect("Time went backwards")
+        .as_secs();
+
+    let sys = System::new();
+    init(&sys);
+    let lt = sys.get_program(1);
+
+    let mut _players = BTreeMap::<u32, Player>::new();
+    let _lottery_history = BTreeMap::<u32, ActorId>::new();
```

Comment:
No need of starting underscore if you use them.
```suggestion
    let mut players = BTreeMap::<u32, Player>::new();
    let lottery_history = BTreeMap::<u32, ActorId>::new();
```